### ISSUE TITLE
Error with node-libs-browser peerDependency

### ISSUE THREAD

**Issue by muffinresearch** [webpack/webpack#1098]
On travis I was seeing the following message for a project using webpack: 

```
npm ERR! peerinvalid The package node-libs-browser does not satisfy its siblings' peerDependencies requirements!
```
It's caused by the project package.json having node-libs-browser specified at version ^0.5.0 which was added to package.json having installed webpack with `npm install webpack --save-dev`

Now there's a version of node-libs-browser higher than 0.5.0 available, that breaks the peerDependency specified in webpack.

Temporary workaround is to fix the version of node-libs-browser to 0.5.0 as opposed to ^0.5.0.

**Comment by jaraquistain** [comment]
Beat me to it! Same issue here as well.

**Comment by SpaceK33z** [comment]
`node-libs-browser` is now a dependency instead of a peerDependency (as of #1197), so this issue is no longer relevant.

**Comment by muffinresearch** [comment]
@SpaceK33z thanks for the heads-up. Closing as fixed by 232618e
